kcp/controllers/watcher_controller.go

```diff
@@ -250,7 +222,7 @@ func (r *WatcherReconciler) createOrUpdateIstioGwForCR(ctx context.Context, isti
 	if util.IsGWConfigChanged(listenerGateway, r.Config.ListenerIstioGatewayPort) {
```

Comment:
I would rather implement a watch on gateway resource to assess if it changed

Reply:
You're right. The Istio Gateway resource should not change. I'll drop the check 👍